src/css/index.css

```diff
@@ -136,6 +148,75 @@ th {
   opacity: 1;
 }
 
+/*Playlist*/
+.track-list {
+  width: 200px;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.55);
+  color: #eee;
+  z-index: 1;
+  position: absolute;
```

Comment:
I've personally found that pretty much every position you're attempting to achieve on this page can be done without position: absolute. Maybe even look into [css grids](https://cssgrid.io/)?

Reply:
Sure thanks, I'll look into it asap.